test/unit/factorization/test_compute_t_factor.cpp

```diff
@@ -92,8 +92,9 @@ void is_orthogonal(const MatrixLocal<const T>& matrix) {
 
 template <class T>
 std::tuple<dlaf::common::internal::vector<T>, MatrixLocal<T>> computeHAndTFactor(
-    const SizeType k, const MatrixLocal<const T>& v) {
-  const SizeType m = v.size().rows();
+    const SizeType k, const MatrixLocal<const T>& v, GlobalElementIndex v_start) {
+  // PRE: m >= k
```

Comment:
Why not making it more explicit with an `ASSERT_*`?

Reply:
The function is not void. Anyway the assert is present in the parent function